steps/container_steps.py

```diff
@@ -8,6 +8,8 @@
 LOG_FORMAT = '%(asctime)s - %(name)s - %(levelname)s - %(message)s'
 logging.basicConfig(format=LOG_FORMAT)
 
+logger = logging.getLogger("cekit")
```

Comment:
I see you've created a logger rather than relying on the default root logger. I wonder if we should be consistent here and always use the `cekit` logger across all step files rather than having a mix (which is the current situation)?

Reply:
I do agree, if @radtriste is wiling to update the remaining places, otherwise we can do this in a separate PR.

Reply:
Problem with the default root logger is that it was not logging anything ...
So decided to create logger

Reply:
reverted the logger

Reply:
@radtriste I think we're talking at cross-purposes - I didn't have any issues with the logger change - rather that it should be consistent thoughout as @spolti said ; i.e. *all* files use `logging.getLogger("cekit")` rather than root logger.